src/transformers/models/esm/__init__.py

```diff
@@ -0,0 +1,71 @@
+# flake8: noqa
+# There's no way to ignore "F401 '...' imported but unused" warnings in this
+# module, but to preserve other warnings. So, don't check this module at all.
+
+# Copyright 2020 The HuggingFace Team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from typing import TYPE_CHECKING
+
+from ...utils import OptionalDependencyNotAvailable, _LazyModule, is_torch_available
+
+
+_import_structure = {
+    "configuration_esm": ["ESM_PRETRAINED_CONFIG_ARCHIVE_MAP", "ESMConfig"],
+    "tokenization_esm": ["ESMTokenizer"],
+}
+
+try:
+    if not is_torch_available():
+        raise OptionalDependencyNotAvailable()
+except OptionalDependencyNotAvailable:
+    pass
+else:
+    _import_structure["modeling_esm"] = [
+        "ESM_PRETRAINED_MODEL_ARCHIVE_LIST",
+        "ESMForCausalLM",
+        "ESMForMaskedLM",
+        "ESMForSequenceClassification",
+        "ESMForTokenClassification",
+        "ESMLayer",
```

Comment:
Given Sylvain's comments above, I would likely not include the `ESMLayer` here as well

Reply:
Done!